compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1362,6 +1366,8 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
         );
 
         if lookup_ident.span.at_least_rust_2018() {
+            // It will be sorted later.
```

Comment:
I wrote this because of your statement above but I couldn't find where it is sorted.

Reply:
If it's not clear whether it's sorted and where, then it needs to be sorted here.
Performance-wise it doesn't matter much because we are in diagnostic code, and extern prelude is already cloned here.

Reply:
Addressed by https://github.com/rust-lang/rust/pull/131213/commits/93ab85f3ad250661d6e6d0107a0b7d44c75f9392
Please check.